apps/judicial-system/backend/src/app/messages/requestPdf.ts

```diff
@@ -1,27 +1,6 @@
 import { defineMessage, defineMessages } from '@formatjs/intl'
 
 export const restrictionRequest = {
-  caseType: defineMessages({
```

Comment:
Isn't it dangerous to delete these strings in this PR? What happens when the changes are pushed to CF? Will that break prod until a new release is deployed?

I don't have the answer to this question. If deleting the strings poses a problem, then you may consider creating a new cleanup task that can be worked on after this PR is on prod.

Reply:
Yes you are right and good point, I will do a separate task for removing these strings.